.github/workflows/compliance.yml

```diff
@@ -121,10 +120,12 @@ jobs:
         if [ "${exit}" == "1" ]; then
           echo "Compliance error, check for error messages in the \"Run Compliance Tests\" step"
           echo "You can run this step locally with the ./scripts/ci/check_compliance.py script."
+          echo "exit_code=${exit}" >> $GITHUB_OUTPUT
           exit 1;
         fi
-
-        if [ "${{ steps.pr_description.outcome }}" == "failure" ]; then
-          echo "PR description cannot be empty"
-          exit 1;
-        fi
+    - name: resolve
+      if: |
+        steps.check-warns.outputs.exit_code != 0 &&
+        ! contains(github.event.pull_request.labels.*.name, 'Compliance: False Positive')
```

Comment:
Maybe we should make the `PR Metadata Check` workflow fail when a PR is labeled `Compliance: False Positive` so that a manual override is necessary; otherwise, this can be quite easily abused ...